packages/fast-components-styles-msft/src/slider-label/index.ts

```diff
@@ -0,0 +1,135 @@
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { applyFocusVisible, toPx } from "@microsoft/fast-jss-utilities";
+import { SliderLabelClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { glyphSize, height, horizontalSpacing } from "../utilities/density";
+import { neutralForegroundActive, neutralOutlineRest } from "../utilities/color";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    applyLocalizedProperty,
+    Direction,
+    ellipsis,
+} from "@microsoft/fast-jss-utilities";
+import { applyCornerRadius, applyFocusPlaceholderBorder } from "../utilities/border";
+import { applyCursorDefault, applyCursorPointer } from "../utilities/cursor";
+import { applyDisabledState } from "../utilities/disabled";
+import { applyScaledTypeRamp } from "../utilities/typography";
+
+const styles: ComponentStyles<SliderLabelClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<SliderLabelClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        sliderLabel: {
+            display: "grid",
+        },
+
+        sliderLabel_positioningPanel: {
+            display: "grid",
+        },
+
+        sliderLabel_label: {
+            ...applyScaledTypeRamp("t9"),
+            color: neutralForegroundActive,
+        },
+
+        sliderLabel_tickMark: {
+            background: neutralOutlineRest,
+        },
+
+        sliderLabel__positionMin: {},
+
+        sliderLabel__positionMax: {},
+
+        sliderLabel__orientationHorizontal: {
+            "&$sliderLabel": {
+                alignSelf: "start",
+                gridRow: "2",
+                marginTop: "-2px",
```

Comment:
Generally we should avoid negative margins can this be achieved another way

Reply:
I believe this is one of those "no better choice" scenarios.  Adding additional elements to be the rounded endcaps would work but doesn't seem better to me.